Userland/Libraries/LibJS/Runtime/Temporal/Instant.h

```diff
@@ -22,6 +22,7 @@ class Instant final : public Object {
     virtual ~Instant() override = default;
 
     [[nodiscard]] BigInt const& nanoseconds() const { return m_nanoseconds; }
+    [[nodiscard]] BigInt& nanoseconds() { return m_nanoseconds; }
```

Comment:
Doesn't have to be in this PR, but it looks like the only reason we ever use non `const&` BigInt is for constructing Instant/ZonedDateTime themselves (i.e., the constructor takes a non-constant ref and the member is a non-constant ref), so we can just replace both of those with `const&` as well and use only `const&` everywhere.